lib/path.js

```diff
@@ -439,57 +439,17 @@ const win32 = {
     if (len === 0)
       return false;
     var code = path.charCodeAt(0);
-    if (len > 1) {
-      if (code === 47/*/*/ || code === 92/*\*/) {
-        // Possible UNC root
-
-        code = path.charCodeAt(1);
-        if (code === 47/*/*/ || code === 92/*\*/) {
-          // Matched double path separator at beginning
-          var j = 2;
-          var last = j;
-          // Match 1 or more non-path separators
-          for (; j < len; ++j) {
-            code = path.charCodeAt(j);
-            if (code === 47/*/*/ || code === 92/*\*/)
-              break;
-          }
-          if (j < len && j !== last) {
-            // Matched!
-            last = j;
-            // Match 1 or more path separators
-            for (; j < len; ++j) {
-              code = path.charCodeAt(j);
-              if (code !== 47/*/*/ && code !== 92/*\*/)
-                break;
-            }
-            if (j < len && j !== last) {
-              // Matched!
-              last = j;
-              // Match 1 or more non-path separators
-              for (; j < len; ++j) {
-                code = path.charCodeAt(j);
-                if (code === 47/*/*/ || code === 92/*\*/)
-                  break;
-              }
-              if (j !== last)
-                return true;
-            }
-          }
-        }
-      } else if ((code >= 65/*A*/ && code <= 90/*Z*/) ||
-                 (code >= 97/*a*/ && code <= 122/*z*/)) {
-        // Possible device root
-
-        code = path.charCodeAt(1);
-        if (path.charCodeAt(1) === 58/*:*/ && len > 2) {
-          code = path.charCodeAt(2);
-          if (code === 47/*/*/ || code === 92/*\*/)
-            return true;
-        }
-      }
-    } else if (code === 47/*/*/ || code === 92/*\*/) {
+    if (code === 47/*/*/ || code === 92/*\*/) {
       return true;
+    } else if ((code >= 65/*A*/ && code <= 90/*Z*/) ||
```

Comment:
Right, and it would be a breaking change anyway. OK, all questions addressed - LGTM.